Master fails on style with the latest `flake8`

```
tianqi:pfio$ flake8 pfio
pfio/__init__.py:68:48: F821 undefined name 'IOBase'
pfio/io.py:19:55: F821 undefined name 'IOBase'
pfio/io.py:82:47: F821 undefined name 'IOBase'
pfio/io.py:90:64: F821 undefined name 'IOBase'
pfio/io.py:109:52: F821 undefined name 'IOBase'
pfio/containers/zip.py:119:47: F821 undefined name 'IOBase'
pfio/containers/zip.py:127:64: F821 undefined name 'IOBase'
pfio/filesystems/hdfs.py:180:47: F821 undefined name 'IOBase'
pfio/filesystems/hdfs.py:188:64: F821 undefined name 'IOBase'
tianqi:pfio$ flake8 --version
3.8.2 (mccabe: 0.6.1, pycodestyle: 2.6.0, pyflakes: 2.2.0) CPython 3.7.4 on
Linux
tianqi:pfio$ git branch
  bump_v_1_0_0
  bump_version_0_1_2
* master
  pr_114
  pr_121
  pr_125
  refactor_zip_test
  rename_to_pfio
tianqi:pfio$  
```

--- a/pfio/__init__.py
+++ b/pfio/__init__.py
@@ -1,3 +1,4 @@
+from io import IOBase
 from pfio._context import DefaultContext
 from pfio.version import __version__  # NOQA
 

--- a/pfio/containers/zip.py
+++ b/pfio/containers/zip.py
@@ -2,6 +2,7 @@ from pfio.container import Container
 from pfio.io import FileStat
 from pfio.io import open_wrapper
 import io
+from io import IOBase
 import logging
 import os
 import sys

--- a/pfio/filesystems/hdfs.py
+++ b/pfio/filesystems/hdfs.py
@@ -6,6 +6,7 @@ from krbticket import KrbTicket, SingleProcessKrbTicketUpdater
 import subprocess
 import re
 import io
+from io import IOBase
 import logging
 import os
 import getpass

--- a/pfio/io.py
+++ b/pfio/io.py
@@ -2,6 +2,7 @@ import abc
 import stat
 from abc import abstractmethod
 from importlib import import_module
+from io import IOBase
 
 from pfio.profiler import IOProfiler
 

